### PR TITLE
fixed text deploy-model-using-flask-and-heroku.md

### PR DESCRIPTION
**Pull request by josemoracard**

### CODE REVIEW

File: 07-ml_deploy/deploy-model-using-flask-and-heroku.md [hunk 1]
```diff
@@ -367,48 +363,48 @@ if __name__=="__main__":
...
+You should already have an account on Heroku, but if you don't, go ahead and create your account at https://www.heroku.com.
 
 Let's make sure we also have the following before deploying to Heroku:
 
-1. Gunicorn handles requests and takes care of complicated things. Download gunicorn to your virtual environment. You can use pip to download it.
+1. Gunicorn handles requests and takes care of complicated things. Download `gunicorn` to your virtual environment. You can use `` to download it.
 
 ```bash
-pip install gunicorn
+ install gunicorn
```
Comment (tommygonzaleza) [inline 07-ml_deploy/deploy-model-using-flask-and-heroku.md:377]: You removed pip

File: 07-ml_deploy/deploy-model-using-flask-and-heroku.md [hunk 2]
```diff
@@ -367,48 +363,48 @@ if __name__=="__main__":
...
 
 ## Deployment using Heroku
 
-You should already have an account on Heroku, but if you don't, go ahead and create your account at 'https://www.heroku.com'.
+You should already have an account on Heroku, but if you don't, go ahead and create your account at https://www.heroku.com.
 
 Let's make sure we also have the following before deploying to Heroku:
 
-1. Gunicorn handles requests and takes care of complicated things. Download gunicorn to your virtual environment. You can use pip to download it.
+1. Gunicorn handles requests and takes care of complicated things. Download `gunicorn` to your virtual environment. You can use `` to download it.
```
Comment (tommygonzaleza) [inline 07-ml_deploy/deploy-model-using-flask-and-heroku.md:374]: You removed pip

**Review by tommygonzaleza — CHANGES_REQUESTED** [review]

File: 07-ml_deploy/deploy-model-using-flask-and-heroku.md (hunk 1 above)
Comment (josemoracard) [reply 07-ml_deploy/deploy-model-using-flask-and-heroku.md]: done

File: 07-ml_deploy/deploy-model-using-flask-and-heroku.md (hunk 2 above)
Comment (josemoracard) [reply 07-ml_deploy/deploy-model-using-flask-and-heroku.md]: done

**Pull request merged**